rgw/v2/tests/s3_swift/resuables.py

```diff
@@ -65,7 +66,22 @@ def upload_object(s3_object_name, bucket, TEST_DATA_PATH, config, user_info, app
     if object_uploaded_status is None:
         log.info('object uploaded')
 
+def upload_object_with_tagging(s3_object_name, bucket, TEST_DATA_PATH, config, user_info, obj_tag, append_data=False, append_msg=None):
 
+    log.info('s3 object name: %s' % s3_object_name)
+    s3_object_path = os.path.join(TEST_DATA_PATH, s3_object_name)
+    log.info('s3 object path: %s' % s3_object_path)
+    s3_object_size = config.obj_size
+    if append_data is True:
+        data_info = manage_data.io_generator(s3_object_path, s3_object_size, op='append',
+                                             **{'message': '\n%s' % append_msg})
+    else:
+        data_info = manage_data.io_generator(s3_object_path, s3_object_size)
+    if data_info is False:
+        TestExecError("data creation failed")
+    log.info('uploading s3 object: %s' % s3_object_path)
+    print(bucket.put_object(Key=s3_object_name, Body=s3_object_path, Tagging=obj_tag))
```

Comment:
it's a bad idea to use a `print`. Use logging.  
Though right now log uses print, we need to change this in the future. :)

Reply:
Have removed the print function here

Reply:
Can you verify again. Have removed the print function.